### ISSUE TITLE
Responding to TRC Calls to Action, thinking about decolonization and settler-colonialism

### ISSUE THREAD

**Issue by dcwalk** [ournetworks/2019#110]
Just a place to gather the various threads we've highlighted in meetings, etc... around this:

- "As we have questions about how to make space for livelihoods, solidarity, and survival it feels particularly important to also grappling with the settler colonial context of Canada and think about self-determination and nation-building."
- "We also are thinking about how to decentre dominant narratives and voices from conversations about technology, as part of our events we have opened with a land acknowledgement but also thought about how we can respond to the TRC's calls to action throughout the event. We welcome any thoughts on that and are actively forming our approach."
- Areas: (well... how do we weave this through everything?)
  - Opening 
  - Keynote 
  - non-English languages (attempted for presentations, option for presenters but did not happen... what do we need to resource this+translation better?)
  - Zine library
  - How we plan this conference 
  - MC / facilitation -> https://github.com/ournetworks/2019/issues/57
  - Everything else?

**Comment by dcwalk** [comment]
I did some [TRC calls to action](http://trc.ca/assets/pdf/Calls_to_Action_English2.pdf) reflecting, as it has been awhile since I reviewed, I think the aspects that speak to us as an organization fall in the areas of situating the histories we are presenting...

From (Business and Reconciliation) 92. "Provide education for management and staff on the history of Aboriginal peoples, including the history and legacy of residential schools, the United Nations Declaration on the Rights of Indigenous Peoples, Treaties and Aboriginal rights, Indigenous law, and Aboriginal–Crown relations. This will require skills based training in intercultural competency, conflict resolution, human rights, and anti-racism." 
From (Newcomers to Canada) 93. "...reflect a more inclusive history of the diverse Aboriginal peoples of Canada, including information about the Treaties and the history of residential schools."

Places to think about this with:
- https://technologycouncil.ca/
   - "Digital Equity - The digital divide refers to the inequity in internet access between some communities, mainly rural, remote and on-reserve, and other communities, mainly urban and off-reserve, that prevents socioeconomic party between Indigenous and non-Indigenous communities and excludes many Indigenous peoples from the digital economy." 
- Technology and Reconciliation talk: https://sucanada.org/event/technology-and-reconciliation-gabrielle-scrimshaw/
- https://newsinteractives.cbc.ca/longform-single/beyond-94 to see imperfect representation of how many have not been started :(
